docs/nodes/nodes-library/nodes/Mailgun/README.md

```diff
@@ -0,0 +1,39 @@
+---
+permalink: /nodes/n8n-nodes-base.mailgun
+---
+
+# Mailgun
+
+[Mailgun](https://www.mailgun.com/) is a developer-friendly email sending platform that provides API-based email services that are easy to use.
+
+::: tip 🔑 Credentials
+The instructions for authenticating with Mailchimp are [here](../../../credentials/Mailgun/README.md).
+:::
+
+## Basic Operations
+
+- Send an email
+
+## Example Usage
+
+This workflow allows you to send an email using Mailgun. You can also find the [workflow](https://n8n.io/workflows/522) on this website. This example usage workflow uses the following two nodes.
+
+- [Start](../../core-nodes/Start)
+- [Mailgun]()
+
+The final workflow should look like the following image.
+
+![A workflow with the Mailgun node](./workflow.png)
+
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+### 2. Mailgun Node
```

Comment:
node should not be capitalized here (I saw that it has been capitalized in Mailchimp too). Can you please check to make sure that everything is consistent?